### ISSUE TITLE
On Samsung Nexus, panels are white on tap (offset issue ?)

### ISSUE THREAD

**Issue by nylnook** [ajam/pulp#38]
Hi,

I tested Terms of service (http://projects.aljazeera.com/2014/terms-of-service/) with my Samsung Nexus mobile, and with the android browser (Android version 4.1.2 ) page view is fine, but taping on panels will display only a white background, as if the zooming didn't get well and we are looking in the margin.

I'm not sure if the Terms of service pulp instance is up to date, or if I should update my mobile ...

I know Android browser is cheating on it's resolution, so maybe zooming calculation goes wrong because of this cheat...

Thanks in advance !

**Comment by mhkeller** [comment]
We have a Samsung Nexus here, let me see what version I tested it on and get back to you. Does the browser you used have a name or version number to compare?

**Comment by nylnook** [comment]
I searched for a version number, but I'm afraid I couldn't find any... 
According to caniuse.com, Android browser is linked to the system until v4.4 (then it become independant and is named Chome)...
So I suppose my version is 4.1.2, as the system ;p

**Comment by nylnook** [comment]
Doesn't sound good : http://slides.com/html5test/the-android-browser/#/ :disappointed:

**Comment by mhkeller** [comment]
Our Samsung has moved to the other office but with that link in mind, it seems like getting the exact setup that you do looks like a slim possibility. I'll ask around if people have older androids I could use for testing but if upgrading your phone is a possibility, that might be the quickest route.

**Comment by mhkeller** [comment]
@nylnook Are you still seeing this one or has an OS update hopefully resolved it?

**Comment by nylnook** [comment]
Hi !
Sorry, it's still occuring... I haven't updated anything neither...
Again, give me one month !
Thanks ;)

**Comment by nylnook** [comment]
Hi @mhkeller !
As promised, I checked this... 
I have a good news, and a bad news: On Nexus S, the latest and final Android Version is 4.1.2, and will not be updated... So this bug can't be solve on  this device/platerform couple.

But I flashed my phone to install Firefox OS... And Pulp work like a charm with it's browser.

I'm not sure Android 4.1.2 support is really a target for you, but I'll let you decide that.

For the Docpad optimisation, I'll go back to you later ;)

Cheers !
